app/components/events/index.hbs

```diff
@@ -0,0 +1,26 @@
+<Header />
+<main>
+	<section class="event-details">
+		<h2 class="event-upcoming">Upcoming Event</h2>
+		<h1 class="event-title">{{this.event.title}}</h1>
+		<h3 class="event-date-time">{{this.event.date}}</h3>
+		<RegisterSeat
+			@initial="Register & grab your seat!"
+			@link={{this.meeting.link}} />
+	</section>
+	<h2 class="speakers-title">Sessions</h2>
+	<div class="session-wrapper">
+		{{#each this.speakerData as |data| }}
+		<Events::SpeakerDetails
+			@topicName={{data.topic.name}}
+			@speakerName={{data.name}}
+			@links={{data.links}}
+			@info={{data.info}}
+			@details={{data.topic.details}}/>
+		{{/each}}
+	</div>
```

Comment:
I would suggest making a different component for the session-card

Reply:
Done! Created new component with name speaker-card.js